website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -5,17 +5,31 @@ id: "scim"
 sidebar: "Set up SCIM"
 ---
 
-# Set up SCIM <Lifecycle status="beta" />
+# Set up SCIM <Lifecycle status="beta, enterprise" />
 
 The System for Cross-Domain Identity Management (SCIM) makes user data more secure and simplifies the admin and end-user lifecycle experience by automating user identities and groups. You can create or disable user identities in your Identity Provider (IdP), and SCIM will automatically make those changes in near real-time downstream in dbt Cloud.
 
+## Prerequisites 
+
+To configure SCIM in your dbt Cloud environment:
+- You must be on an Enterprise plan.
+- You must be using Okta as your SSO provider.
+- You must have proper permissions to configure the account settings in [dbt Cloud](/docs/cloud/manage-access/enterprise-permissions) and alter app settings in [Okta](https://help.okta.com/en-us/content/topics/security/administrators-admin-comparison.htm).
+- If you have IP restrictions enabled, you must add Okta's IPs to your allowlist.
+
 ### Supported features
 
 The currently available supported features for SCIM are:
 
 - User provisioning and de-provisioning
 - User profile updates
 - Group creation and management
+- Importing groups and users
+
+When users are provisioned, the following attributes are supported
+- Username
```

Comment:
do we want to use the user attributes instead? e.g. `userName`, `givenName`, `familyName`? or is that for the api?